client/devdocs/design/typography.jsx

```diff
@@ -58,6 +79,104 @@ export default class Typography extends React.PureComponent {
 							there are some lowercase alphabets to which…
 						</p>
 					</Card>
+
+					<h3>How to use:</h3>
+
+					<p>
+						The <code>$serif</code> Sass variable will output the serif font stack.
+					</p>
+
+					<pre>
+						<code>
+							.design__typography-serif &#123;
+							{ '\n\t' }font-family: $serif;
+							{ '\n' }&#125;
+						</code>
+					</pre>
+
+					<h2>Code Typography</h2>
+
+					<p>We use monospace fonts for code blocks, sized at 15px.</p>
+					<p>
+						<code>
+							Monaco, Consolas, "Andale Mono", "DejaVu Sans Mono", "Courier 10 Pitch", Courier,
+							monospace
+						</code>
+					</p>
+
+					<h3>How to use:</h3>
+
+					<p>
+						The <code>$code</code> Sass variable will output the monospaced font stack.
+					</p>
+
+					<pre>
+						<code>
+							.design__typography-code &#123;
+							{ '\n\t' }font-family: $code;
+							{ '\n\t' }font-size: 15px;
+							{ '\n' }&#125;
+						</code>
+					</pre>
+
+					<h2>Typographic Modular Scale</h2>
```

Comment:
Rather than including this section I think we should link to the brand guide.

Reply:
The modular scale info? Why?

(Filippo and I just talked about removing it from the brand guide, since it seems more technical/less high-level, but I'm interested to get your take!)

Reply:
Because this is mostly Calypso-specific info (I guess it _could_ be used in the apps), I think it should live in devdocs.

Reply:
I just thought it shouldn't be in both places.

The problem with adding it to Calypso is that it also applies to other things, like WPCOM...